pkg/rslog/debug_test.go

```diff
@@ -0,0 +1,189 @@
+package rslog
+
+// Copyright (C) 2025 by Posit Software, PBC.
+
+import (
+	"slices"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+// Copyright (C) 2025 by Posit Software, PBC.
```

Comment:
```suggestion
```

Reply:
Ugh. It was in the wrong place, failed the linter, then I duplicated it. So awesome.